reports/report_builder.py

```diff
@@ -3032,23 +3032,19 @@ def ws_98(self, ws):
     
     def ws_100(self, ws):
         counts_list = []
+        secondary_counts = OrderedDict()
         for media_types, models in SHEET_MEDIA_GROUPS:
-            counts = Counter()
             for media_type, model in models.items():
-                for major_topic, topic_ids in GROUP_TOPICS_MAP.items():
-                    rows = model.objects\
-                            .values('covid19')\
-                            .filter(topic=major_topic)\
-                            .annotate(n=Count('id'))
-                    for r in rows:
-                        # Get media id's to assign to counts
-                        media_id = [media[0] for media in media_types if media[1] == media_type][0]
-                        counts.update({(media_id, int(major_topic)): r['n']})
-            counts_list.append(counts)
+                counts = Counter()
+                rows = model.objects\
+                        .values('covid19', 'topic')\
+                        .annotate(n=Count('id'))
+                for r in rows:
+                    covid19 = 'Y' if r['covid19'] == 1 else 'N'
+                    counts.update({(covid19, TOPIC_GROUPS[r['topic']]): r['n']})
+                secondary_counts[media_type] = counts
     
-        self.tabulate(ws, counts_list[0], TM_MEDIA_TYPES, MAJOR_TOPICS, row_perc=True)
-        c = ws.dim_colmax + 2
```

Comment:
We no longer do this littler traditional / digital media columns separation?

Reply:
Naah. `tabulate_secondary_cols` takes care of that.